src/pages/app-bar/search-bar/index.tsx

```diff
@@ -209,7 +209,19 @@ export const SearchBar = (): JSX.Element => {
                     />
                 ))}
                 {list.length < 1 && (
-                    <InfoListItem icon={<Error />} title={'0 results'} subtitle={'No matching presidents'} />
+                    <div
+                        style={{
+                            display: 'flex',
+                            flexDirection: 'column',
+                            height: `calc(100vh - ${theme.spacing(8)}px)`,
+                        }}
+                    >
+                        <EmptyState
+                            icon={<Error fontSize={'inherit'} />}
+                            title={'0 results'}
+                            description={'No matching presidents'}
+                        />
+                    </div>
```

Comment:
Can we take this logic outside the `<List>`, and add some inline comments to it?

Reply:
What kind of inline comments do you suggest?

Reply:
Now that I think twice about it... maybe a comment is not necessary :P

Reply:
Looking good otherwise